Riot/Assets/en.lproj/Vector.strings

```diff
@@ -619,6 +619,7 @@ Tap the + to start adding people.";
 "settings_labs_enable_ringing_for_group_calls" = "Ring for group calls";
 "settings_labs_enabled_polls" = "Polls";
 "settings_labs_enable_threads" = "Threaded messaging";
+"settings_labs_use_only_latest_profiles" = "Use only latest profiles";
```

Comment:
In a similar vein here, this could be a bit clearer and probably worth matching something pretty similar to web. They used this which also calls out where it changes:
> Show current avatar and name for users in message history

[here](https://github.com/matrix-org/matrix-react-sdk/pull/7815/files#diff-20527b4ae184e85263f5a70fc00d1cb022b054ea126e404332da46983247e5bcR887)